src/security/tests/cli_security_tests.c

```diff
@@ -379,6 +397,8 @@ main(void)
 			test_request_credentials_fails_if_reply_token_missing),
 		SECURITY_UTEST(
 			test_request_credentials_fails_if_reply_cred_missing),
+		SECURITY_UTEST(
+			test_request_credentials_fails_if_reply_verifier_missing),
```

Comment:
(style)  line over 80 characters

Reply:
IMO it's fine to abbreviate the test name if it's butting against the 80 char limit.